css/index.css

```diff
@@ -1,3 +1,361 @@
+    border: 1px solid black;
+    background-color: #ececec;
+
+}
+main img:last-of-type{
+    position: relative;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+}
+section:last-of-type{
+    border-bottom: 1px solid;
+}
+.col-1{
+    display: flex;
+}
+.col-2{
+    display: flex;
+}
+.col-top{
+    width: 49%;
+    margin: 0 auto;
+}
+footer{
+    padding: 30px 0;
+}
+.service-footer{
+    padding-top: 10px;
+    display: flex;
+    justify-content: space-between;
+}
+.service-footer ul{
+    list-style: square;
+    padding-top: 20px;
+    padding-left: 20px;
+}
+.service-footer li{
+    padding-bottom: 10px;
+}
+.service-footer div p{
+    padding-right:35px;
+}
+.service-footer .copyright{
+    display: block;
+    text-align: center;
+    padding-top: 50px;
+}
+footer .copyright{
+    display: block;
+    text-align: center;
+    padding-top: 95px;
+}
+
+@media(max-width: 800px){
+    .container {
+        position: relative;
+        width:785px;
+        padding: 50px 0px 50px 0px;
+        border: none;
+        margin: 0 auto;
+    }
+    .main-nav {
+        width: 100%;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+    }
+    .main-nav ul{
+        display: flex;
+        width: 100%;
+        justify-content: space-evenly;
+    }
+    .logo {
+        width: 183px;
+        margin-bottom: 33px;
+    }
+    .main-nav ul :nth-child(1) {order: 1;}
+    .main-nav ul :nth-child(2) {order: 2;}
+    .main-nav ul :nth-child(3) {order: 3;}
+    .main-nav ul :nth-child(4) {order: 4;}
+    .main-nav ul :nth-child(5) {order: 5;}
+    .main-nav ul :nth-child(6) {order: 6;}
+
+    .main-nav :nth-child(1) { order: 2;}
+    .main-nav :nth-child(2) { order: 1;}
+
+    .banner img {
+        display: none;
+    }
+    .banner h1{
+        margin-top: 15px;
+        margin-bottom: 25px;
+    }
+    .banner{
+        margin-bottom: 70px;
+    }
+    section{
+        margin-left: 20px;
+        margin-right: 20px; 
+        padding-top: 35px; 
+
+    }
+    .col-1 p {
+        width: 96%;
+    }
+    .col-2 p{
+        width: 87%;
+    }
+    .middle-img {
+        width: 95%;
+    }
+    footer{
+        padding-left: 30px;
+        padding-top: 35px;
+    }
+    footer .copyright{
+        display: block;
+        text-align: center;
+        padding-top: 35px;
+    }
+ 
+}
+@media(max-width: 500px){
+    .container {
+        position: relative;
+        width:485px;
+        padding: 20px 0px 50px 0px;
+        border: none;
+        margin: 0 auto;
+    }
+
+    .main-nav ul {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        border-top: 1px solid gray;
+        padding: 0;
+    } 
+    .main-nav li {
+        width: 100%;
+        text-align: center;
+        border-bottom: 1px solid gray;
+        padding-top: 21px;
+        padding-bottom: 20px;
+        font-size: 14pt;
+    }
```

Comment:
I noticed that you navigation items after your mobile-view breakpoint are off-centered. This is because your li had a padding-right on it. If you set `padding: 0;` above your `padding-top` and `padding-bottom` parameters, then you are effectively clearing all of the li's padding, and then only setting its top and bottom padding.